plugins/permission-backend-module-policy-allow-all/src/index.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2023 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The allow all policy backend module for the permission plugin.
+ *
+ * @packageDocumentation
+ */
+
+export { permissionModuleAllowAllPolicy } from './module';
```

Comment:
Should this be exported as alpha to align with `@backstage/plugin-permission-backend`, although it wouldnt have an index?

Reply:
I think this one is fine to leave as non-alpha